e2e/test/scenarios/embedding/embedding-dashboard.cy.spec.js

```diff
@@ -283,6 +283,15 @@ describe("scenarios > embedding > dashboard parameters", () => {
       assertRequiredEnabledForName({ name: "User", enabled: false });
       assertRequiredEnabledForName({ name: "Not Used Filter", enabled: false });
     });
+
+    it("should render cursor pointer on hover over a toggle", () => {
```

Comment:
Nit: please add issue reference to the title.

Reply:
there's no issue associated with this one - this is just a fix from [this slack thread](https://metaboat.slack.com/archives/C064QMXEV9N/p1722040565625579) - I'll just use #46223 ?